libs/agora/api-description/src/api.openapi.yaml

```
@@ -30,11 +30,11 @@ paths:
   /biodomains:
     $ref: paths/biodomains.yaml
   /biodomains/{ensg}:
-    $ref: paths/biodomains/@{ensg}.yaml
+    $ref: paths/biodomains/ensg.yaml
```

The curled brackets should be conserved to indicate path parameters.
```suggestion
    $ref: paths/biodomains/{ensg}.yaml
```

---

The same comment applies to other path variables.